### DISCUSSION TITLE
PLEASE SOMEONE HELP ME. After i say to Monika my eyes color in aparience dialogue the game crash with a error

### DISCUSSION THREAD

**Discussion by AnnieLovesMonika** [Monika-After-Story/MonikaModDev#9671] [QA - tech support + general] [answered]
I saw that another person had the same problem, but the difference is that I don't have any in-game submods and it's the M.A.S (Monika After Story) and DDLC mod. I did some research but I don't know what the error is and even trying to find the files I can't find a way to fix it! Please help me I do
n't want to lose all the progress :( **(ERROR NAME: AFM_PREF NBOT DEFINED)**
![image](https://user-images.githubusercontent.com/116119035/196539523-f5205b5c-4946-4b3d-889a-e84a0e9b03c7.png)

**Comment by AnnieLovesMonika** [comment]
> I saw that another person had the same problem, but the difference is that I don't have any in-game submods and it's the M.A.S (Monika After Story) and DDLC mod. I did some research but I don't know what the error is and even trying to find the files I can't find a way to fix it! Please help me I do n't want to lose all the progress :( **(ERROR NAME: AFM_PREF NBOT DEFINED)** ![image](https://user-images.githubusercontent.com/116119035/196539523-f5205b5c-4946-4b3d-889a-e84a0e9b03c7.png)

I installed all in DDLC oficial page and the mod too, i don't khow what is happening-

**Comment by Booplicate** [comment, marked as answer]
You're using severely outdated version of the mod. Update to `0.12.11`.

**Reply by AnnieLovesMonika** [reply]
tyy! i will do that!
